onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1152,53 +1098,114 @@ void addObjectMethods(py::module& m, Environment& env, ExecutionProviderRegistra
       sess(m, "SessionOptions", R"pbdoc(Configuration information for a session.)pbdoc");
   sess
       .def(py::init())
-      .def_readwrite("enable_cpu_mem_arena", &PySessionOptions::enable_cpu_mem_arena,
-                     R"pbdoc(Enables the memory arena on CPU. Arena may pre-allocate memory for future usage.
+      .def_property(
+          "enable_cpu_mem_arena",
+          [](const PySessionOptions* options) -> bool { return options->value.enable_cpu_mem_arena; },
+          [](PySessionOptions* options, bool enable_cpu_mem_arena) -> void {
+              options->value.enable_cpu_mem_arena = enable_cpu_mem_arena;
```

Comment:
Note that `PySessionOptions` is now an alias of `OrtSessionOptions`, which contains a member called `value` of type `onnxruntime::SessionOptions`.